pkgs/applications/blockchains/electrs.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, llvmPackages
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "electrs";
+  version = "0.8.6";
+
+  src = fetchFromGitHub {
+    owner = "romanz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0nnblxz4xr8k083wy3whx8qxqmdzbxsh5gd91161mrnvidganvgb";
+  };
+
+  # needed for librocksdb-sys
+  nativeBuildInputs = [ llvmPackages.clang ];
+  LIBCLANG_PATH = "${llvmPackages.libclang}/lib";
+
+  cargoSha256 = "11xwjcfc3kqjyp94qzmyb26xwynf4f1q3ac3rp7l7qq1njly07gr";
+
+  meta = with lib; {
+    description = "An efficient re-implementation of Electrum Server in Rust";
+    homepage = "https://github.com/romanz/electrs";
+    license = licenses.mit;
+    maintainers = with maintainers; [ prusnak ];
```

Comment:
Please add `meta.platforms`.

Reply:
Addressed in 479ffac739fdaab3d791aa9aeae7ea8133616b71

Reply:
If platforms is set to platforms.unix than it is not required for buildRustPackage, buildGoModule or buildPythonPackage because they all default to it.

Reply:
Reverted in 488c4cf2e50fad49efd9602196e9fa34fac895c1